app/src/main/res/drawable/general_item_background_border_grey.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+  android:shape="rectangle">
+  <solid android:color="@color/component_color_general_item_background_solid_color" />
+  <stroke
+    android:width="1dp"
+    android:color="@color/component_color_general_item_secondary_background_stroke_color" />
+
+</shape>
```

Comment:
EOF is missing, add one blank line at the end.

Reply:
Applied, i was confused about the blank line but now I have corrected PTAL, Thanks